python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -17,16 +17,21 @@
 SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "#provider-alert")
 SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
 SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
+IS_RUNTIME_RELEASE = bool(os.getenv("IS_RUNTIME_RELEASE", False))
 
 
 def get_report(dag_run_ids: List[str], **context: Any) -> None:  # noqa: C901
     """Fetch dags run details and generate report"""
+
     with create_session() as session:
         last_dags_runs: List[DagRun] = session.query(DagRun).filter(DagRun.run_id.in_(dag_run_ids)).all()
         message_list: List[str] = []
 
         airflow_version = context["ti"].xcom_pull(task_ids="get_airflow_version")
-        airflow_version_message = f"Airflow version for the below astro-sdk run is `{airflow_version}` \n\n"
+        if IS_RUNTIME_RELEASE:
+            airflow_version_message = f"Results generated for latest Runtime version {os.environ['ASTRONOMER_RUNTIME_VERSION']} with {os.environ['AIRFLOW__CORE__EXECUTOR']}  \n\n"
+        else:
+            airflow_version_message = f"Airflow version for the below astro-sdk run is `{airflow_version}` with {os.environ['AIRFLOW__CORE__EXECUTOR']} \n\n"
```

Comment:
How is Airflow installed in this scenario? I'm not familiar with the Turbulence approach, and since it is a private repo, it would be worth giving meaningful information as part of this PR.

Do we use another docker container, built with the latest version of Airflow. What is this latest version of Airflow (nightly build, main..)?

Reply:
@tatiana Below are the steps that turbulence performs to create a airflow deployment.

1. Make a new deployment on a cloud stage environment with GraphQL calls.
2. Creates a dockerfile with the latest runtime image(alpha, stable),latest astro-sdk, example dags and we build this dockerfile and create a image hash.
3. Use Astro CLI to update the newly created deployment(in #1) with the image has created (in #2)

Reply:
Thanks, @vatsrahul1001 ! Is the latest Astro SDk the one in `main` or the last released Astro SDK?

Reply:
@tatiana we are using last released